fairlearn/postprocessing/plotting.py

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Utilities for plotting curves."""
+
+from ._constants import _MATPLOTLIB_IMPORT_ERROR_MESSAGE, DEMOGRAPHIC_PARITY, EQUALIZED_ODDS
+from ._threshold_optimizer import ThresholdOptimizer
+
+try:
+    import matplotlib.pyplot as plt
+    import matplotlib.cm as cm
+    import matplotlib.colors
+
+    tab10_norm = matplotlib.colors.Normalize(vmin=0, vmax=7)
+    tab10_scalarMap = cm.ScalarMappable(norm=tab10_norm, cmap='Dark2')
+    debug_colors = [tab10_scalarMap.to_rgba(x) for x in range(10)]
+    debug_ncolors = len(debug_colors)
+    debug_colormap = {}
+
+    highlight_color = [0.95, 0.90, 0.40]
+except ImportError:
+    raise RuntimeError(_MATPLOTLIB_IMPORT_ERROR_MESSAGE)
```

Comment:
This only happens when someone explicitly runs something from this file, and not just when they `import fairlearn.postprocessing` ?

Reply:
Yes! In fact we are importing fairlearn.postprocessing earlier in this file

Reply:
So long as it's severable :-)